run_all_tests.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-sbt clean compile coverage test it:test coverageOff coverageReport
\ No newline at end of file
+sbt compile coverage test it:test coverageOff coverageReport
```

Comment:
hooray :)

Reply:
Clean was doing my nut in haha

Reply:
Yep. I changed this locally the other day too.

Reply:
feel free to change it for the other microservices as well